units/event_metajob.go

```diff
@@ -319,7 +321,8 @@ func (j *eventMetaJob) logEventCount(eventCount, limit int) error {
 		"job_id":  j.ID(),
 		"job":     eventMetaJobName,
 		"message": "unprocessed event count",
-		"count":   eventCount,
+		"pending": eventCount,
+		"current": len(j.events),
```

Comment:
We're already logging this as `eventCount` above.

Reply:
different.